packages/blockchain-wallet-v4-frontend/src/layouts/Wallet/MenuTop/Balance/BtcWatchOnlyBalance/index.js

```diff
@@ -18,6 +18,10 @@ class BtcWatchOnlyBalance extends React.PureComponent {
     this.props.actions.fetchData()
   }
 
+  componentDidUpdate () {
+    this.props.data.getOrElse() && this.props.hasBtcSubBalance()
+  }
```

Comment:
This seems a bit backwards. It tells the parent component about a boolean, when the parent component itself could know

Reply:
I guess fine

Reply:
The parent can't know about it because of where the selector is located. I'll move things around to make this more clear.